### PR TITLE
delete component

### PR DESCRIPTION
**Pull request by RanjanaMalode**

### CODE REVIEW

File: src/main/java/com/iop/productservice/controller/ComponentController.java [hunk 1]
```diff
@@ -6,12 +6,7 @@
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
-import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.PathVariable;
-import org.springframework.web.bind.annotation.PostMapping;
-import org.springframework.web.bind.annotation.RequestBody;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.*;
```
Comment (khandaresachin) [inline src/main/java/com/iop/productservice/controller/ComponentController.java]: remove this * from the import and just type **ctrl+alt+o**  for rearranging the imports

File: src/main/java/com/iop/productservice/service/ComponentService.java [hunk 2]
```diff
@@ -41,4 +41,10 @@ public ResponseEntity<Component> createComponent(ComponentRequest request) {
         }
         return new ResponseEntity<>(created, HttpStatus.CREATED);
     }
+
+    public ResponseEntity<String> deleteComponent(Long componentId) {
+        getComponent(componentId);
```
Comment (khandaresachin) [inline src/main/java/com/iop/productservice/service/ComponentService.java:49]: writedown some loggers in service implementation method

File: src/main/java/com/iop/productservice/controller/ComponentController.java (hunk 1 above)
Comment (RanjanaMalode) [reply src/main/java/com/iop/productservice/controller/ComponentController.java]: implementation done

File: src/main/java/com/iop/productservice/service/ComponentService.java (hunk 2 above)
Comment (RanjanaMalode) [reply src/main/java/com/iop/productservice/service/ComponentService.java:49]: implementation done

**Review by khandaresachin — APPROVED** [review]
PR changes are approved

**Pull request merged**